### PR TITLE
upgraded to scala play 2.8 MISUV-2222

### PR DESCRIPTION
**Pull request by joesheridan**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -8,19 +8,20 @@ import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
...
 def test(scope: String = "test,it"): Seq[ModuleID] = Seq(
-  "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "5.12.0" % scope,
+  "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "5.13.0" % scope,
   "org.pegdown" % "pegdown" % "1.6.0" % scope,
   "org.jsoup" % "jsoup" % "1.11.3" % scope,
   "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
-  "org.scalatestplus.play" %% "scalatestplus-play" % "4.0.3" % scope,
-  "org.mockito" % "mockito-core" % "2.7.22" % "test"
+  "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % scope,
+  "org.scalatestplus" %% "mockito-3-4" % "3.2.9.0" % scope,
```
Comment (MihailsC-hmrc) [inline build.sbt:23]: 👍

File: project/plugins.sbt [hunk 2]
```diff
@@ -6,7 +6,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.4.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.9")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7")
```
Comment (MihailsC-hmrc) [inline project/plugins.sbt:9]: Note: the latest version is `2.8.8` at the moment.

Comment (joesheridan) [reply project/plugins.sbt:9]: ok - want me to update it?

File: build.sbt [hunk 3]
```diff
@@ -8,19 +8,20 @@ import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
...
+  "com.vladsch.flexmark" % "flexmark-all" % "0.35.10" % scope
```
Comment (MihailsC-hmrc) [inline build.sbt:24]: Note: the latest version is `0.62.2` at the moment.

**Review by MihailsC-hmrc — CHANGES_REQUESTED** [review]
Please include JIRA story number in the PR title. And address any feasible comments.

File: project/plugins.sbt (hunk 2 above)
Comment (MihailsC-hmrc) [reply project/plugins.sbt:9]: @joesheridan Up to your discretion. Probably better to update.

File: build.sbt (hunk 3 above)
Comment (joesheridan) [reply build.sbt:24]: 0.62.2 is not compatible with the scalatest/mockito versions

File: project/plugins.sbt (hunk 2 above)
Comment (joesheridan) [reply project/plugins.sbt]: ok done

**Review by MihailsC-hmrc — APPROVED** [review]
LGTM

**Pull request merged**